tests/acceptance/test_project_settings_sampling.py

```diff
@@ -50,9 +59,12 @@
 }
 
 
+@pytest.mark.snuba
+@requires_snuba
 class ProjectSettingsSamplingTest(AcceptanceTestCase):
     def setUp(self):
         super().setUp()
+        self.now = datetime(2013, 5, 18, 15, 13, 58, 132928, tzinfo=pytz.utc)
```

Comment:
@ahmedetefy ... @andriisoldatenko found out that the issue was with this `datetime`, since it is `patched` in the `AcceptanceTestCase` class and that is why the API was not returning the groups we needed